Added necessary files for contributors

dhruvapte26 opened the pull request:
Reference to Issue #45 
Created a:
- Code of conduct
- Pull request template
- Issue template
- contributing

dhruvapte26 committed:
Created code-of-conduct

dhruvapte26 committed:
Create PULL_REQUEST_TEMPLATE.md

dhruvapte26 committed:
Created issue template

dhruvapte26 committed:
created a contributing.md

athityakumar commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,19 @@
+2. Enhance the code?
+3. Solve bugs?
+
+We heartily welcome you! :tada:
```
A ❤️  would be more apt here than 🎉 here , I guess.

athityakumar commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,19 @@
+3. Make your changes in the branch
+4. Commit your changes and send a pull request
+
+And :tada:! You have contributed to this cool project!
```
-> And you've contributed to this cool project! 🎉

athityakumar commented on ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,11 @@
+## Do you wish a TV series?
```
-> Do you wish ***to add*** a TV Series?

athityakumar commented:
Thanks a lot for working on this, @the-ethan-hunt. There are a few broad changes I'd like to add on -

1. Can we shift the "Working of the script" part to Contribution Guidelines?
2. Please refer to [this repo](https://github.com/athityakumar/colorls) for "better" Contribution Guidelines, Issue / PR templates and link them as required from the README.
3. Add yourself as a maintainer on the README, will you? 😄

dhruvapte26 committed:
Updated changes as per maintainer suggestions

dhruvapte26 committed:
corrected typo

dhruvapte26 committed:
Transferred workflow to CONTRIBUTING.md

dhruvapte26 committed:
Transferred workflow

dhruvapte26 committed:
rewrote PR template

dhruvapte26 commented:
@athityakumar , thanks for your insights and reviews on this. :smile: 
I have made the necessary changes.
>Add yourself as a maintainer on the README, will you? :smile: 

I am sorry,  but I did not understand this. Do I need to mention my name at the end of the `readme.md`? :sweat_smile:

athityakumar commented:
@the-ethan-hunt - Yep, mention your name as a maintainer anywhere in the README. 😄

athityakumar commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,40 @@
+Firstly, thanks for contributing to the project. :smile:
```
`# Contribution Guidelines` as the top-most section.

athityakumar commented:
Just one minor change as suggested above, and we're good to merge. 😄

dhruvapte26 committed:
Added title

dhruvapte26 committed:
Mentioned maintainers

dhruvapte26 commented:
@athityakumar , I have added both of us in the maintainer section. I saw this on various good repositories that even the owner is mentioned as a maintainer. I have added our GitHub profiles as the links. Do tell me if you want your blog as the link. :smile:
